Web/WM03/README.md

```diff
@@ -5,7 +5,45 @@
 > Visit the site at https://cfta-wm03.allyourbases.co and find a way to bypass the password check.
 
 ## Solution
+Going to the source page, we can see that the logic for the login is as below:
+```
```

Comment:
Use ```html for syntax highlighting

Reply:
got it, thanks!

Reply:
fixed in https://github.com/Alic3C/Cyber-FastTrack-Spring-2021/pull/4/commits/2d29456bc21f106fda13f6e6fe940a7b6804a33f